spectrum/checks.py

```diff
@@ -568,6 +568,15 @@ def just_load(self, path):
         _assert_status_code(response, 200, url)
         return response
 
+    def redirect(self, path, expected, status_code=301):
+        url = _build_url(path, self._host)
+        LOGGER.info("Loading %s", url)
+        response = requests.get(url, allow_redirects=False)
+        _assert_status_code(response, status_code, url)
+        location = response.headers['Location']
+        assert location.startswith(self._host)
+        assert location == ('%s%s' % (self._host, expected))
```

Comment:
`_host` = scheme + authority?

Reply:
Yes, is there a better name for `https://elifesciences.org`, `https://end2end--cdn-elifesciences.org`?

Reply:
'[Origin](https://html.spec.whatwg.org/multipage/origin.html#origin)' is the nearest formal term I'm aware of, but 'base URI' might do.

Reply:
Consistently inaccurate name will require a consistent update in many places